docs/user/kafka/layer_views.rst

```diff
@@ -0,0 +1,37 @@
+.. _kafka_layer_views:
+
+Layer Views
+-----------
+
+A common use case for the Kafka data store is to expose a single layer with different default filters
+and relational transforms to different users. GeoMesa provides a way to do this through configuring
+views on top of the original layer.
+
+To enabled layer views, use the ``kafka.layer.views`` data store parameter when creating a store:
+
+.. code-block:: java
+
+    import org.geotools.data.DataStore;
+    import org.geotools.data.DataStoreFinder;
+
+    String views =
+      "{" +
+      "  type1 = [" +
+      "    { type-name = \"transformView\", transform = [ \"dtg\", \"geom\", \"name\" ] }," +
+      "    { type-name = \"filterView\", filter = \"bbox(geom,0,0,10,10)\" }" +
+      "  ]," +
+      "  type2 = [" +
+      "    { type-name = \"filterTransformView\", filter = \"bbox(geom,0,0,10,10)\", transform = [ \"dtg\", \"geom\" ] }" +
```

Comment:
What order are the transforms and filters applied in?  Can one filter on a new / transformed column?

Hmm... actually, this doesn't show a new column being computed.  Is that possible?

Reply:
we always apply filters first and then transforms, i think mainly b/c you're allowed to filter on things that are not in the transform. it should be possible to use any of the normal transforms we use in our queries, it's the same code that does the transform, although I don't have an example of it here.

Reply:
Always?  In the stats DSL, I remember working to setup transforms first so that they could be filtered and have stats computed on new columns.

I can't find a unit test proving that quickly enough though...

Reply:
pretty sure it's always filter first

Reply:
Pretty sure I did transform, filter, aggregate...  I know aggregate is last.;)  I'll have to look sometime...